examples/kind/kind-expose-port.yaml

```diff
@@ -2,7 +2,6 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-- role: worker
```

Comment:
Is this change intentional?

Reply:
I was looking for ways to reduce resource usage in the integration tests, so my thought was taking out this worker would help and I think it has.

Reply:
This isn't used though, did you mean `_integration/testsuite/kind-expose-port.yaml`?